evalml/pipelines/components/transformers/column_selectors.py

```diff
@@ -167,6 +166,20 @@ def _check_input_for_columns(self, X):
     def _modify_columns(self, cols, X, y=None):
         return X.ww.select(cols)
 
+    def fit(self, X, y=None):
+        """Fits the transformer by checking if column names are present in the dataset.
+
+        Args:
+            X (pd.DataFrame): Data to check.
+            y (pd.Series, optional): Targets.
```

Comment:
Hyper-nit: are we indicating y as "ignored" here?

Reply:
Will edit here and above as well.